pkg/minishift/util/proxy_util_test.go

```diff
@@ -23,7 +23,8 @@ import (
 func TestParseProxyUriWithHttpProxy(t *testing.T) {
```

Comment:
FYI, in Golang one often uses table driven tests when the test setup is so similar - https://dave.cheney.net/2013/06/09/writing-table-driven-tests-in-go

Maybe next time